src/test/java/com/iexec/standalone/result/ResultServiceTests.java

```diff
@@ -293,6 +297,71 @@ void testSaveResultInfo() {
         assertThat(resultInfo.getDatasetUri()).isEqualTo("datasetUri");
     }
 
+    @Test
+    void testGetResultModelWithZip() throws IOException {
+        ResultInfo resultInfo = mock(ResultInfo.class);
+        when(resultInfo.getImage()).thenReturn("image");
+        when(resultInfo.getCmd()).thenReturn("cmd");
+        when(resultInfo.getDeterministHash()).thenReturn("hash");
+
+        doReturn(resultInfo).when(spyResultService).getResultInfos(CHAIN_TASK_ID);
+        try (MockedStatic<Files> mockedFiles = mockStatic(Files.class)) {
```

Comment:
Using static mocks is usually not a best practice when you can do otherwise. In the current case, you should have a look at the `@TempDir` annotation which create a temporary folder for you to write test files in it.